gwcs/coordinate_frames.py

```diff
@@ -115,10 +115,8 @@ def _set_axis_physical_types(self, pht=None):
                     ph_type.append("custom:{}".format(axt))
                 else:
                     ph_type.append(axt)
-            validate_physical_types(ph_type)
-            return tuple(ph_type)
 
-        if isinstance(self, CelestialFrame):
+        elif isinstance(self, CelestialFrame):
```

Comment:
I'm not sure I understand the use case for this change. In the original, if physical_type (pht) is passed it is assumed it covers all axes and is simply returned. If it's not passed the code iterates over the axis and assigns default values.

Reply:
I moved the return to the end, to remove the duplication of 
```
            validate_physical_types(ph_type)
            return tuple(ph_type)
```